.gitlab/functional_test/system_probe.yml

```diff
@@ -12,6 +12,7 @@
   rules:
     !reference [.on_system_probe_changes_or_manual]
   stage: functional_test
+  timeout: 3h
```

Comment:
A trigger to a bigger problem - for me, that's a major concern
the pipelines are already slow enough (kitchens tests are starting very soon, but as well we're the slowest job as well).
Can we run each OS on a different job?

Reply:
I asked agent-platform about that setup, and they gave approval so I will do that in a different PR.

Reply:
I went ahead and made that change in this PR